<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionWindow</class>
 <widget class="QMainWindow" name="VersionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>VNC</string>
    </property>
    <widget class="QPushButton" name="buttonRemoveVnc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuń</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonUpdateVnc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Aktualizuj</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRestoreVnc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Przywróc</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBackupVnc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Kopia</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="labelVncStatus_2">
      <property name="text">
       <string>USB</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelVncUsb">
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="labelVncStatus">
      <property name="text">
       <string>Zainstalowano</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelVncInstalled">
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>161</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>HYBRYDA</string>
    </property>
    <widget class="QPushButton" name="buttonRemoveHybryda">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuń</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonUpdateHybryda">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Aktualizuj</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonResotreHybryda">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Przywróc</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBackupHybryda">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Kopia</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="labelVncUsb_2">
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="labelHybrydaStatus_2">
       <property name="text">
        <string>USB</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelHybrydaUsb">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="labelVncInstalled_2">
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="labelHybrydaInstalled">
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelHybrydaStatus">
       <property name="text">
        <string>Zainstalowano</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>345</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>STATUS
 DYSKU</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDriveStatus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Read only</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Odblokuj</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
